lib/main_common.pm

```diff
@@ -516,6 +516,9 @@ sub load_extra_tests {
     }
     else {
         loadtest "console/zypper_lr_validate";
+        if (is_sle && sle_version_at_least('15')) {
```

Comment:
Why would it only apply for SLE15?

Reply:
because SLE 15 uses Roles and related repositories which is not the case for openSUSE other older SLE.